vantagev1/swagger.json

```diff
@@ -7746,39 +7742,6 @@
         "operationId": "getCosts"
       }
     },
-    "/integrations/terraform/workspaces/{workspace_id}/{integration_type}": {
```

Comment:
what's the background on what happened to these other endpoints?

Reply:
That is a good question

Reply:
provider tests pass locally? possibly someone checked in something erroneously. `terraform` in the url is sus - to the `git blame`! 🦇

Reply:
Added two years ago :huh: https://github.com/vantage-sh/vantage-go/commit/c2baa44f08fc39455bba3538d514a62f9a22e822

Reply:
Ok, Brooke just removed it: https://github.com/vantage-sh/core/commit/5d6d03304bbb33aa64b301a696fd0b3652f0d330